sources/engine/Stride.Games/Desktop/GameForm.cs

```diff
@@ -371,7 +382,7 @@ protected override void WndProc(ref Message m)
                         //also remove full screen if this is the case
                         if (IsFullScreen && !isSwitchingFullScreen) //exit full screen on alt-tab if in fullscreen
                         {
-                            OnFullscreenToggle(new EventArgs());
```

Comment:
Could users take a dependency on this event being invoked? If so we shouldn't remove it.

Reply:
It was inherintly broken, this should only ever be false when hitting this if statement if I understood correctly. 

This is a mandatory state change when exiting fullscreen which requires it to be false or else it will reset to true due to the toggle logic and cause a crash.

Reply:
My point here is that if someone had some custom logic in a listener to the Toggle event, by removing this invocation their logic would never be invoked. If the whole event is to be made obsolete let's make sure to add an attribute to the event so the users start getting a warning there.

Reply:
Although I may have misunderstood - if this isn't the main place where the toggle callback was invoked then ignore my comment.

Reply:
It is not the main place where it is used. It was meant to change the fullscreen to not fullscreen when the window was being changed but due to how broken the windowing stuff is it just gets called randomly. With this intance being always false it wont be toggled back to true randomly.

There is another case of `OnFullscreenToggle` being used correctly below when alt tabbing. I still think this logic is wrong but its at least functionally correct.
```
                case Win32Native.WM_SYSKEYDOWN: //alt is down
                    if (wparam == VK_RETURN)
                    {
                        isSwitchingFullScreen = true;
                        if (!enableFullscreenToggle) return;
                        OnFullscreenToggle(new EventArgs()); //we handle alt enter manually
                        isSwitchingFullScreen = false;
                    }
                    break;
```